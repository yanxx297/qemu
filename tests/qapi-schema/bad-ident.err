tests/qapi-schema/bad-ident.json:2: 'type' does not allow optional name '*oops'
