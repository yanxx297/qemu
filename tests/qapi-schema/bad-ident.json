# we reject creating a type name with bad name
{ 'type': '*oops', 'data': { 'i': 'int' } }
