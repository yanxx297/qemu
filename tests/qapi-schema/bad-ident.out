[OrderedDict([('type', '*oops'), ('data', OrderedDict([('i', 'int')]))])]
[]
[OrderedDict([('type', '*oops'), ('data', OrderedDict([('i', 'int')]))])]
